docs/.gitignore

```
@@ -18,12 +18,13 @@ report-check-links.xlsx
 /.vuepress/public/handsontable
 /.vuepress/public/@handsontable
 /.vuepress/public/data/
+!/.vuepress/dist
 /.build-tmp/
 
 /docker/redirects-autogenerated.conf
 /docker/redirect-page-ids.json
 /docker/variables.conf
-/content/api/
+# /content/api/
```

```suggestion
/content/api/
```
The same as with `/.vuepress/dist`. It should be ignored in the `develop` and feature branches.

---

Fixed. content/api is ignored